CommandViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace CnCNet.LauncherStub;
+
+using System.Windows.Input;
+
+public class CommandViewModel : NotifyPropertyChangedBase
+{
+    private string? text;
+
+    public string Text
+    {
+        get => text; set
```

Comment:
Put the setter on its own line, looks weird to have it after the semicolon of the getter.